game/events.yaml

```diff
@@ -1046,13 +1046,393 @@
     offset: P4MT58M24S
     duration: P1M
 
+- eventID: "flood-hits-new-york"
+  title: "Too much rain in the last couple of days has led to big floods in New York"
+  description: >-
+    "Due to the mass in rain, the city has flooded with millions of property damage.
+    Many of New Yorks most prestigious buildings and firms are affected and have claimed to suffer damages in the high millions"
+
+  Type: "random"
+  randomChancePerDay: 0.009
+
+  duration: P1W
+
+  dependencies: []
+  effects: []
+
+  tags:
+    - affectsTags: ["financial"]
+      minTrend: -1.6
+      maxTrend: -1
+      affectsStockIDs: ["JPM","AXP", "GS"]
+
+
+- eventID: "hurricane-hits-caribbean"
+  title: "big-hurricane-in-the-caribbean-holds-health-services-in-check"
+  description: >-
+    "A big hurricane that originated south-east of the Florida peninsula, has been going on a rampage through many islands in the Caribbean.
+    The health organisations are struggling to keep supply flowing to those in need"
+
+  Type: "random"
+  randomChancePerDay: 0.009
+
+  duration: P3W
+
+  dependencies: []
+  effects: []
+
+  tags:
+    - affectsTags: ["health"]
+      minTrend: -1.6
+      maxTrend: -1
+      affectsStockIDs: ["DVA","IDEXX","ANTM"]
+
+
+- eventID: "low-oil-levels"
+  title: "new-research-suggests-that-we-are-running-out-of-oil"
+  description: >-
+    "New Research from Harvard University who partnered with Yale University has concluded that we will run out of oil sooner, rather than later."

```

Comment:
```suggestion
    "New research from Harvard University who partnered with Yale University has concluded that we will run out of oil sooner, rather than later."
```